DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Bindings/RhinoBasicConnectorBinding.cs

```diff
@@ -74,12 +71,12 @@ public void HighlightModel(string modelCardId)
 
     if (myModel is SenderModelCard sender)
     {
-      objectIds = sender.SendFilter.GetObjectIds();
+      objectIds = sender.SendFilter.NotNull().GetObjectIds().ToList();
```

Comment:
Why the `.ToList`? this list is going to be very large. we should try and avoid unnecessary allocations